test/utils/getPossibleFilePaths.test.js

```diff
@@ -1,12 +1,18 @@
-import getFilenameFromUrl from '../../src/utils/getFilenameFromUrl';
+import getFilenameFromUrl from '../../src/utils/getPossibleFilePaths';
```

Comment:
I will rewrite these tests to real tests in the next PR to ensure all works fine with express too